File: 4- partida de futebol/futebol.c
#include <stdio.h>

int main() {
    int golsTime1, golsTime2;
    printf("Digite os gols do time 1: ");
    scanf("%d", &golsTime1);
    printf("Digite os gols do time 2: ");
    scanf("%d", &golsTime2);

    if (golsTime1 > golsTime2) {
        printf("Time 1 venceu!\n");
    } else if (golsTime2 > golsTime1) {
        printf("Time 2 venceu!\n");
    } else {
        printf("Empate!\n");
    }
    return 0;
}

File: 4- partida de futebol/futebol.py
gols_time1 = int(input("Digite os gols do time 1: "))
gols_time2 = int(input("Digite os gols do time 2: "))
if gols_time1 > gols_time2:
    print("Time 1 venceu!")
elif gols_time2 > gols_time1:
    print("Time 2 venceu!")
else:
    print("Empate!")
